examples/core/radix-name-service/src/lib.rs

```diff
@@ -0,0 +1,256 @@
+use scrypto::prelude::*;
+use sha2::{Digest, Sha256};
+
+#[derive(NftData)]
+struct DomainName {
+    #[scrypto(mutable)]
+    address: Address,
+    #[scrypto(mutable)]
+    last_valid_epoch: u64,
+    #[scrypto(mutable)]
+    deposit_amount: Decimal,
```

Comment:
Is mutable `deposit_amount` intentional? I didn't find anywhere updating this critical field.

Reply:
No it isn't. I was originally going to increase the deposit amount on name renewal, but then decided to make it a non-refundable fee.